ehitchhiking-ui/src/app/shared/modals/profile-modal/profile-modal.component.ts

```diff
@@ -13,12 +12,8 @@ import {CarsInfoService} from './cars-info.service';
 })
 export class ProfileModalComponent implements OnInit {
 	readonly maxNumOfCars: number = 5;
```

Comment:
It is a constant value hence feel free to use capitalize letters in it's name: MAX_NUMBER_OF_CARS